lib/conjur/conjur_config.rb

```diff
@@ -23,6 +23,8 @@ class ConjurConfig < Anyway::Config
     attr_config(
       # Read TRUSTED_PROXIES before default to maintain backwards compatibility
       trusted_proxies: (ENV['TRUSTED_PROXIES'] || []),
+      users_authorization_token_ttl: 480, # default is 8 minutes
+      hosts_authorization_token_ttl: 480, # default is 8 minutes
```

Comment:
Please change these to the singular.  It will improve the readability of these settings.
```suggestion
      user_authorization_token_ttl: 480, # default is 8 minutes
      host_authorization_token_ttl: 480, # default is 8 minutes
```

Reply:
@oburstein-hub, please add unit tests to check that the authorization token expires when expected.

Reply:
@jvanderhoof  - please move your comments to the JIRA Ticket

Reply:
done + added UT to cover all cases